tower_cli/utils/resource_decorators.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Ansible by Red Hat
+# Alan Rominger <arominge@redhat.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+from tower_cli.utils import types
+
+import click
+
+
+def unified_job_template_options(method):
+    """
+    Adds the decorators for all types of unified job templates,
+    and if the non-unified type is specified, converts it into the
+    unified_job_template kwarg.
+    """
+    jt_dec = click.option(
```

Comment:
Try:

`tower-cli node list --project examples`

and you'll get all the nodes that run the project called "examples". This is still a "lie", but a good one.